ir/src/main/scala/org/scalajs/core/ir/Printers.scala

```diff
@@ -681,10 +681,19 @@ object Printers {
 
         case IntLiteral(value) =>
           if (value >= 0) {
-            print(value)
+            print(value.toString)
           } else {
             print('(')
-            print(value)
+            print(value.toString)
+            print(')')
+          }
+
+        case LongLiteral(value) =>
+          if (value >= 0) {
+            print(value.toString)
+          } else {
+            print('(')
+            print(value.toString)
             print(')')
```

Comment:
Appending an `L` would append the value rather than the string representation. Couldn't it?